src/pages/index.js

```diff
@@ -38,6 +41,11 @@ const IndexPage = ({
       ></SubjectButton>
       <ActionButton>Categories</ActionButton>
       <SkillsMapButton />
+      <WorkplaceExamples
+        title="Workplace Examples"
+        body="some text some more text"
```

Comment:
this will need one piece of data: workplaceExamples which will be an array of strings. or you could pass in 4 props but I think it would be neater in an array